src/common/components/mapView/MapView.tsx

```diff
@@ -45,10 +46,32 @@ type Props = {
   items: MapItem[];
   center?: LatLngExpression;
   zoom?: number;
+  focusedItemId?: string;
 };
 
-function MapView({ items = [], center, zoom }: Props) {
+function MapView({ items = [], center, zoom, focusedItemId }: Props) {
```

Comment:
With this API change the `MapView` component entangles more business logic. It's something I'm okay with, but personally I would've attempted to keep that away from this component insofar as it's possible. E.g. provide the zoom as a prop and open the marker in a handler.

It's a little bit of extra work and I'm not entirely sure how it would work out so this approach may be more pragmatic in the end, which is why I think it to be a good choice as well.